tests/ctests/test_vector_index_stats.c

```diff
@@ -1,6 +1,15 @@
 #include "test_util.h"
 #include "info/vector_index_stats.h"
 
+int test_memory_and_marked_deleted_invalid_input() {
+    VectorIndexStats_Getter getter = VectorIndexStats_GetGetter("invalid");
+    ASSERT(getter == NULL);
+
+    VectorIndexStats_Setter setter = VectorIndexStats_GetSetter("invalid");
+    ASSERT(setter == NULL);
```

Comment:
why is it still not covered?

Reply:
Solved by @Itzikvaknin  in [3fe4d6d](https://github.com/RediSearch/RediSearch/pull/5480/commits/3fe4d6d2b1dfd1ac13f127cfd50ba256af3b1d00)